web/virtual_lab/static/virtual_lab/js/chemistry/precipitation.js

```
@@ -0,0 +1,134 @@
+// Draw beaker outline
+function drawBeaker() {
+  bctx.clearRect(0,0,600,400);
+  bctx.strokeStyle = '#333';
+  bctx.lineWidth = 3;
+  bctx.strokeRect(200,100,200,250);
+}
+
+// Swirl animation
+function drawSwirl() {
+  sctx.clearRect(0,0,600,400);
+  const cx=300, cy=300, r=100;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)